shared/oae/macros/list.html

```diff
@@ -332,7 +332,7 @@
     {/if}
 
     {if profilePath && displayOptions.addLink !== false}
-        <h3 {if displayOptions.threedotTitle !== false} class="oae-threedots"{/if}><a href="${profilePath|profilePath}" title="${entityData.displayName|encodeForHTMLAttribute}" {if displayOptions.linkTarget} target="${displayOptions.linkTarget}"{/if}>${entityData.displayName|encodeForHTML}</a></h3>
```

Comment:
This title attribute isn't entirely redundant. As threedotting is applied here, the tooltip is the only way in which the full title can be seen.